sample-apps/jaeger-zipkin-sample-app/build.gradle

```diff
@@ -0,0 +1,52 @@
+plugins {
+    id 'java'
+
+    // Apply the application plugin to add support for building a CLI application.
+    id 'application'
+    id 'com.google.cloud.tools.jib' version "2.7.1"
```

Comment:
I think you didn't use jib in the config ... and we are not using jib when building image in terraform as well .. https://github.com/aws-observability/aws-otel-test-framework/blob/09fdedd1c80f2fbe8fbdc91787f612a5cea93535/terraform/imagebuild/main.tf#L59

It seems @anuraaga introduced jib, is it only used for local test?

Reply:
Jib pushes the test framework image for use in other repos without building it every time.

https://www.github.com/aws-observability/aws-otel-test-framework/tree/terraform/.github%2Fworkflows%2Fmain-build.yml

We don't need to bring up any infrastructure for running tests for SDKs, just run on GitHub runner, so it's easy to run them with docker-compose

Reply:
ok, so you only need the validator and do not need the sample apps? https://github.com/aws-observability/aws-otel-test-framework/issues/228